internal/utils/errors.go

```diff
@@ -0,0 +1,6 @@
+package utils
+
+const (
```

Comment:
This can go into errors.

Reply:
I did not add it there following Nandi comments that he would not want to duplicate those big motherf*** lines. ;)

Reply:
stop thinking to change fasthttp.. the implementation of it is different from other HTTP libraries..

Reply:
@nandiheath no it's really not. I don't wanna to do it but if u don't see the value, I might implement another http framework just to show u that both of them can coexist and u can use one or another.

Reply:
what i see here is we got everything in server package (cool). But you still have to rewrite route+controllers to work with the other HTTP package. So at the end of the day, the work is still heavy without true abstraction...

Reply:
if this is a simple request it will be fine to switch to another HTTP package. however since the implementation of fasthttp is very different compared to other packages (to be 10x performant than other), if you are modifying the headers/post body/ etc. logic they cannot be port directly to other packages.

What I am trying to say is, abstraction and logic separation are good, but sometimes if you cannot avoid it just let it be. k8s/mongodb/postgres those I of coz agree to separate, but fasthttp is not the case.